SafeguardSessions/Src/SafeguardSessions.pq

```diff
@@ -90,11 +94,11 @@ shared GetData = (
         result;
 
 ValidateSPSVersion = (
-    spsUrl as text, sessionId as text, skipVersionCheck as logical, optional mocks as nullable record
+    spsUrl as text, sessionId as text, skipVersionCheck as nullable text, optional mocks as nullable record
```

Comment:
In SafeguardSessions.pq all functions are directly tested, except ValidateSPSVersion. Shouldn't we write a tests for this under the Test > Integration folder?

Reply:
Nice catch. I'll extend it with tests.

Reply:
Done.